lib/rules/at-rule-semicolon-newline-after/__tests__/index.js

```diff
@@ -79,42 +79,42 @@ testRule(rule, {
   reject: [
     {
       code: "@mixin foo; a {}",
+      fixed: "@mixin foo;\n a {}",
       message: messages.expectedAfter(),
       line: 1,
       column: 12
     },
     {
       code: "@mIxIn foo; a {}",
+      fixed: "@mIxIn foo;\n a {}",
       message: messages.expectedAfter(),
       line: 1,
       column: 12
     },
     {
       code: "@MIXIN foo; a {}",
+      fixed: "@MIXIN foo;\n a {}",
       message: messages.expectedAfter(),
       line: 1,
       column: 12
     },
     {
       code: '@import url("x.css"); @charset "UTF-8";',
+      fixed: '@import url("x.css");\n @charset "UTF-8";',
       message: messages.expectedAfter(),
       line: 1,
       column: 22
     },
     {
       code: '@charset "UTF-8"; a {};',
-      message: messages.expectedAfter(),
-      line: 1,
-      column: 18
-    },
-    {
-      code: '@charset "UTF-8"; a {};',
+      fixed: '@charset "UTF-8";\n a {};',
       message: messages.expectedAfter(),
       line: 1,
       column: 18
     },
     {
       code: "a{\n@extend .b; @extend .c\n}",
+      fixed: "a{\n@extend .b;\n @extend .c\n}",
```

Comment:
Can we have the same example, but with `\r\n` newlines, please?

Reply:
sure - in addition to this one or as a replacement?

Reply:
In addition, please.